app/MilliMeter.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace ConverterApp;
+
+class MilliMeter extends Length
+{
+    protected $power;
+    public function __construct()
+    {
+        $this->power = -3;
```

Comment:
power is member of Length class so it should be handled by that class. If you create Metric class then that class should handle power.